doc/user-docs/customization.md

```diff
@@ -0,0 +1,65 @@
+# Windows Terminal Customization
+
+NOTE: At the time of writing Windows Terminal is still under active development and many things will
+change. If you notice an error in the docs, please raise an issue. Or better yet, please file a PR with an appropriate update!
+
+## Title of tab
+When Windows Terminal launches, the shell has a special variable to determine the window title. This title is used to set the name of the tab inside Windows Terminal.
+There are 2 options for controlling the title of the tab.
+
+1. Set the title from the settings of Windows Terminal
+2. Set the title within the shell (varies by shell)
+
+#### 1. Set the title from the settings of Windows Terminal
+In the `settings.json` file from Windows Terminal, you can overrule the title of the tab by specifying the property `tabTitle`.
+The `tabTitle` overrules the title from the individual shell. Please note that the `tabTitle` is static.
+
+#### 2. Set the title within the shell (varies by shell)
+This is a more dynamic solution, but may require you to run a command within the shell itself (depending on the shell).
+Below are som example options for controlling the title of your shell.
+
+#### Examples of setting the title within the shell
+##### PowerShell
+```
+$Host.UI.RawUI.WindowTitle = "My PowerShell"
+```
+###### Documentation: [Get-Host - Microsoft Docs][psgethost] | [PSHostRawUserInterface - Microsoft Docs][pswindowtitle]
+
+##### CMD
+```
+title My cmd
+```
+###### Documentation: [title - Microsoft Docs][cmdtitle]
+
+## Customizing the prompt
+The prompt is the part of the terminal shown before where your cursor is placed. 
```

Comment:
Suggest something like: 

"The prompt is the text that is displayed at the start of each command-line, to the left of the cursor. For example, in Cmd, the default prompt is `C:\>`